runtime/queries/php/highlights.scm

```diff
@@ -8,6 +8,14 @@
 (named_type (name) @type) @type
 (named_type (qualified_name) @type) @type
 
+(namespace_definition
+  name: (namespace_name(name)) @namespace
+)
```

Comment:
```suggestion
(namespace_definition
  name: (namespace_name (name) @namespace))
```

The current pattern is ignored because the `((name) @constructor ...` stanza on L56 is more specific.